subsys/bluetooth/services/ias/ias_client.c

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright (c) 2022 Codecoup
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <errno.h>
+#include <init.h>
+#include <stdint.h>
+#include <zephyr.h>
+#include <net/buf.h>
+#include <logging/log.h>
+#include <bluetooth/gatt.h>
+#include <bluetooth/conn.h>
+#include <bluetooth/services/ias.h>
+
+#define LOG_LEVEL CONFIG_BT_IAS_CLIENT_LOG_LEVEL
+LOG_MODULE_REGISTER(ias_client);
+
+#define IS_HANDLE_VALID(handle) ((handle) != 0x0000)
+
+static struct bt_ias_client_cb *ias_client_cb;
+static struct bt_ias_client client_list[CONFIG_BT_MAX_CONN];
+
+static struct bt_ias_client *client_by_conn(struct bt_conn *conn)
+{
+	return &client_list[bt_conn_index(conn)];
+}
+
+int bt_ias_alert_write(struct bt_conn *conn, uint8_t alert_lvl)
+{
+	int err;
+	struct net_buf_simple buf;
```

Comment:
This will not work, as you do not define buffer to store the data, so  `net_buf_simple_init_with_data` will probably override stack

Reply:
Changed it so it does no longer makes use of net_buf_simple.